p2p-specs/p2p-interface.md

```diff
@@ -208,7 +210,7 @@ chainId: '1'
 entryPointContract: '0x0576a174d229e3cfa37253523e645a78a0c91b57'
 description: >-
   This is the default/canonical mempool, which will be used by most bundlers on
-  Ethereum Mainnnet
```

Comment:
I know that it's not part of this PR, but shouldn't we have EntryPoint version somewhere here as well?